django_tables2/utils.py

```diff
@@ -394,7 +399,7 @@ def resolve(self, context, safe=True, quiet=False):
                     if safe and getattr(current, "alters_data", False):
                         raise ValueError(self.ALTERS_DATA_ERROR_FMT.format(method=current.__name__))
                     if not getattr(current, "do_not_call_in_templates", False):
-                        current = current()
+                        current = current(*self.callable_args, **self.callable_kwargs)
```

Comment:
I'm not sure on the `callable_`-prefix, can we simplify this by just calling them `self.args` and `self.kwargs`?

Reply:
I kind of like it being more verbose as it tells the user what those args and kwargs are being used for.  If you disagree, I can make them be args and kwargs no problem.  Another option is to not wild-card them and have the user pass them as a list/dict instead.  I am going to change it to the later and you let me know what you think